db/migrate/20151125155213_create_middleware_servers.rb

```diff
@@ -1,14 +1,17 @@
 class CreateMiddlewareServers < ActiveRecord::Migration
```

Comment:
modifying the existing migration script is problematic because it is not an incremental change - I, for example, have no convenient way to apply this script to my environment's db because I already ran the previous script's version (yes, it's doable by deleting the table and etc., but it's not the right flow :) )